### ISSUE TITLE
Training and Validation Loss

### ISSUE THREAD

**Issue by tengen119** [Maxhin/PyTorch_Malaria_Detection#1]
![Malaria_Tr_val_loss](https://user-images.githubusercontent.com/60868520/95409378-6f11a280-095c-11eb-8c1d-4df9a2446114.png)
